File: Basic/Hangman_game/v1/hangman.c
#include <stdio.h>
/**
 * Hangman game (word guessing game).
 * We create an array of words and
 * ask the user to guess a word we have in the array.
 *
 * main - entry point of the program.
 */

char start_game();
int guessed_result();

int main(void)
{
	char words[4][15] = {"Beauty", "Salutation", "Love", "Developer"};
	start_game();

	//test
	for (int i = 0; i < 4; i++)
	{
		printf("\n%s\n", words[i]);
	}
	return (0);
}

/**
 * start_game - starts the game and welcome players
 *
 */
char start_game()
{
	printf("Welcome to the Hangman game...\n");
	printf("Guess the right word\n");
	printf("You have three guesses\n");
}

File: Basic/Hangman_game/v1/hangman.py
#!/usr/bin/python3

"""

This is the version one of the hangman game in python.
we have a list of words and a user is asked to guess 
which of the words is in our list of words.

"""

# Words to guess from.
words = ['Beauty', 'Salutation', 'Welcome', 'Python', 'Programming']

active = True

#ask user to guess
while active:

    guess = input("Press q to quit game or Type here to play... ")

    for word in words:
        if guess == 'q':
            active = False
        elif guess == word:
            print(f"{word} You saved him!")
            break
        else:
            print(f"Guess again")
            break
